opal/mca/common/cuda/common_cuda.c

```diff
@@ -1718,8 +1715,8 @@ static void cuda_dump_evthandle(int verbose, void *evtHandle, char *str) {
     }
     memcpy(&evtH, evtHandle, sizeof(evtH));
     opal_output_verbose(verbose, mca_common_cuda_output,
-                        "CUDA: %s:ctxId=%d, pid=%d, index=%d",
-                        str, (int)evtH.ctxId, evtH.pid, (int)evtH.index);
+                        "CUDA: %s:pid=%d, serial=%d, index=%d",
+                        str, (int)evtH.pid, (int)evtH.serial, (int)evtH.index);
```

Comment:
These vars are `unsigned long` -- can't they be truncated if you print them as `int`s?  Why is a PID an `unsigned long`, BTW?

If they are really `unsigned long`, then use the appropriate `%` sequence to print `unsigned long` -- don't squash them into an `int`.